src/collective/jsonmigrator/blueprints/source_remote.py

```diff
@@ -71,7 +75,7 @@ def request(self, host, handler, request_body, verbose):
                 "AUTHORIZATION",
                 "Basic %s"
                 % string.replace(
-                    encodestring("%s:%s" % (self.username, self.password)), "\012", ""
+                    encodebytes("%s:%s" % (self.username, self.password)), "\012", ""
```

Comment:
I do not if this is expected to work in Python3.
But I am pretty sure it will fail 3 because it is missing text an bytes multiple times.

It seems to me the proper **Python 3** code should look like:
```
"Basic %s" % encodebytes(("%s:%s" % (self.username, self.password)).encode()).replace(b"\012", b"").decode()
```
This is totally untested.
It should be probably rewritten in some other form to be more readable.